spec/models/custom_button_spec.rb

```diff
@@ -350,14 +350,11 @@
           it "with an array" do
             Timecop.freeze(Time.now.utc) do
               User.with_user(user) { custom_button.send(method, [vm, vm2, vm3], 'UI') }
-              expect(CustomButtonEvent.first.timestamp).to be_within(0.01).of(Time.now.utc)
+              expect(CustomButtonEvent.find_by(:target_id => vm.id, :target_type => "VmOrTemplate", :source => 'UI').timestamp).to be_within(0.01).of(Time.now.utc)
```

Comment:
Does this one matter which record we get? I'm not really sure what is being tested here.